error_report/views.py

```diff
@@ -23,6 +28,9 @@ def v1(request):
     widget_module, colon, lineno = widget_module.rpartition(":")
     module = request.POST.get("Module", "unknown")
     rel_path = _cleanup(os.path.join(widget_module, version, module))
```

Comment:
I would refactor this into a function called get_report_path and call it here.

Reply:
Fixed.